packages/terra-demographics-banner/package.json

```diff
@@ -0,0 +1,62 @@
+{
+  "name": "terra-demographics-banner",
+  "main": "lib/DemographicsBanner.js",
+  "private": true,
+  "version": "0.0.0",
+  "description": "terra-demographics-banner",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/cerner/terra-ui.git"
+  },
+  "keywords": [
+    "Cerner",
+    "Terra",
+    "terra-demographics-banner",
+    "DemographicsBanner",
+    "UI"
+  ],
+  "author": "Cerner Corporation",
+  "license": "Apache-2.0",
+  "bugs": {
+    "url": "https://github.com/cerner/terra-ui/issues"
+  },
+  "homepage": "https://github.com/cerner/terra-ui#readme",
+  "peerDependencies": {
+    "react": "15.4.2",
+    "react-dom": "15.4.2",
+    "terra-mixins": "^1.0.0",
+    "terra-base": "^0.x"
+  },
+  "dependencies": {
+    "classnames": "^2.2.5",
+    "react": "15.4.2",
```

Comment:
I think it's right now. https://github.com/cerner/terra-core/pull/204/commits/d6076dcfd33c6d0bb6d75d1e68bd42edd753aebb